### PR TITLE
Add MFA "UI only" removal blog post

### PR DESCRIPTION
**Pull request by kevinlinxc**
## Summary
Part of [Deprecate ui_only and migrate users to ui_and_gem_signin](https://github.com/rubygems/rubygems.org/issues/2968), an effort to remove the `UI only` MFA level for Rubygems which has been deemed insecure.

Users can no longer change their MFA level to UI Only after https://github.com/rubygems/rubygems.org/pull/3084 was merged about a month ago, and this blog post explains why.

## Dates:
I picked the following dates somewhat arbitrarily and would like to know if they are fine:
July 26th for the blog post date
August 26th for the day of `UI only` -> `UI and gem signin` migration

## Related
When this blog post is out, it could be linked in this guide PR too: https://github.com/rubygems/guides/pull/311

### CODE REVIEW

**Comment by kevinlinxc** [comment]
Preview: 
image

Precedent for having two authors, although it looks weird next to a single portrait: https://github.com/rubygems/rubygems.github.io/blob/master/_posts/2018-02-24-tls-10-and-11-deprecation-notice.md?plain=1#L4

**Review by jenshenny — COMMENTED** [review]
✨ I think a month is reasonable for this change (but I don't have any prior context to say otherwise).

You can leave my name off of the authors imo since this was mainly your work (ie. the blog post and the feature).

File: _posts/2022-07-26-mfa-ui-only-removed.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: An Update to RubyGems MFA Levels
+layout: post
+author: Kevin Lin and Jenny Shen
+author_email: kevin.lin@shopify.com
+---
+
+# Summary
+
+The “UI only” multi-factor authentication level is being removed.
```
[the remark this answers is not in the document]
Comment (kevinlinxc) [reply _posts/2022-07-26-mfa-ui-only-removed.md]: I'll do it to be consistent, but I also kind of think the code style rendering has bad kerning and is harder to read:
image

File: _posts/2022-07-26-mfa-ui-only-removed.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+# Summary
+
+The `UI only` multi-factor authentication level is being removed.
+Users who are currently on the `UI only` MFA level will still remain on that level, but will be migrated to the `UI and gem signin` level on August 26th, 2022.
+
+# Why is this happening?
+
+In essence, the `UI and gem signin` level is the same as `UI only`, but it also ensures that command line logins are verified with multi-factor authentication.
```
Comment (sonalkr132) [inline _posts/2022-07-26-mfa-ui-only-removed.md:15]:
```suggestion
In essence, the `UI and gem signin` level is the same as `UI only`, but it also requires that `gem signin` command is verified with multi-factor authentication. This ensures that accounts with MFA protections are not impacted by compromised passwords.

If you are using `gem sigin` in automated builds, we recommend that you migrate to using a pre-generated API key for authentication. You can either set `API_KEY` env to the value of your API key or write the key to `credentials` file.

echo "https://rubygems.org: YOUR_API_KEY" > ~/.gem/credentials

```

File: _posts/2022-07-26-mfa-ui-only-removed.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+title: An Update to RubyGems MFA Levels
+layout: post
+author: Kevin Lin
+author_email: kevin.lin@shopify.com
+---
+
+# Summary
+
+The `UI only` multi-factor authentication level is being removed.
+Users who are currently on the `UI only` MFA level will still remain on that level, but will be migrated to the `UI and gem signin` level on August 26th, 2022.
```
Comment (sonalkr132) [inline _posts/2022-07-26-mfa-ui-only-removed.md:11]: Let push this to 8 Sept. Please update the date of this post to 8 Aug as well.

Comment (kevinlinxc) [reply _posts/2022-07-26-mfa-ui-only-removed.md:11]: Sorry I missed this, I'll assume that a month of buffer is still desired. I'll put it on the 22nd, next monday.

File: _posts/2022-08-22-mfa-ui-only-removed.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+title: An Update to RubyGems MFA Levels
+layout: post
+author: Kevin Lin
+author_email: kevin.lin@shopify.com
+---
+
+# Summary
+
+The `UI only` multi-factor authentication level is being removed.
+Users who are currently on the `UI only` MFA level will still remain on that level, but will be migrated to the `UI and gem signin` level on September 22nd, 2022.
```
Comment (jchestershopify) [inline _posts/2022-08-22-mfa-ui-only-removed.md:11]: I'd shorten this to just
> Users who remain on the `UI only` MFA level will be migrated to the `UI and gem signin` level on September 22nd, 2022.

File: _posts/2022-08-22-mfa-ui-only-removed.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+# Why is this happening?
+
+In essence, the `UI and gem signin` level is the same as the `UI only` level, but it also requires that the `gem signin` command is verified with multi-factor authentication.
+This ensures that accounts with MFA protection are not impacted by compromised passwords.
+
+# What is going to change?
+After the migration, users that were on the `UI only` level will need to enter a one-time password when using `gem signin`. For everyone else, nothing will change.
```
Comment (jchestershopify) [inline _posts/2022-08-22-mfa-ui-only-removed.md:19]: s/users that/users who/

**Review by jchestershopify — COMMENTED** [review]
Two small tweaks but otherwise LGTM.

**Comment by kevinlinxc** [comment]
@sonalkr132 Do you think this is ready to put out? The date is set for today.

**Review by simi — APPROVED** [review]

**Comment by sonalkr132** [comment]
Awesome. thank you.
